standalone/detectors/fasttrack/include/varstate.h

```diff
@@ -1,15 +1,18 @@
+#ifndef VARSTATE_HEADER_H
+#define VARSTATE_HEADER_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define VARSTATE_HEADER_H
```